components/eamxx/src/physics/rrtmgp/rrtmgp_test_utils.hpp

```diff
@@ -88,7 +88,7 @@ static void dummy_clouds(
   // put them in 2/3 of the columns since that's roughly the total cloudiness of earth.
   // Set sane values for liquid and ice water path.
   // NOTE: these "sane" values are in g/m2!
-  Kokkos::parallel_for( MDRP::template get<2>({nlay,ncol}) , KOKKOS_LAMBDA (int ilay, int icol) {
```

Comment:
Does the new iteration pattern work best for kokkos? That is, if one wants to use YAKL, should they revert back to a lay-col iteration (rather than col-lay)?

Reply:
Same question for all the swaps in the other files below

Reply:
Also, do the percentage changes in your screenshot refer to a yakl/kokkos comparison where both use the same layout? Or did you compare "the best layout for kokkos" against "the best layout for yakl"?

Reply:
Yes, this pattern works best for Kokkos. Note that this is code specific to Kokkos only, so YAKL code is not impacted. Both Kokkos and YAKL should currently be using the patterns that are best for them.